.github/workflows/basic.yml

```diff
@@ -348,6 +348,49 @@ jobs:
         -DUMF_TESTS_FAIL_ON_SKIP=ON
         -DUMF_LINK_HWLOC_STATICALLY=ON
 
+    - name: Build UMF
+      run: cmake --build ${{env.BUILD_DIR}} --config ${{matrix.build_type}} -j $Env:NUMBER_OF_PROCESSORS -v
+
+    - name: Run tests
+      working-directory: ${{env.BUILD_DIR}}
+      run: ctest -C ${{matrix.build_type}} --output-on-failure --test-dir test
+
+  windows-dynamic_mingw_hwloc:
+    name: "Windows dynamic UMF + mingw libhwloc"
+    strategy:
+      matrix:
+        build_type: [Release]
+
+    runs-on: 'windows-2022'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        fetch-depth: 0
+
+    - name: Get hwloc from offitial repo (mingw version)
+      run: |
+        curl https://download.open-mpi.org/release/hwloc/v2.10/hwloc-win64-build-2.10.0.zip -o ${{github.workspace}}\hwloc-win64-build-2.10.0.zip
+        Expand-Archive ${{github.workspace}}\hwloc-win64-build-2.10.0.zip -DestinationPath ${{github.workspace}}
+
+    - name: Get TBB from github
+      run: |
+        curl https://github.com/oneapi-src/oneTBB/releases/download/v2021.12.0/oneapi-tbb-2021.12.0-win.zip -o ${{github.workspace}}\oneapi-tbb-2021.12.0-win.zip
+        Expand-Archive ${{github.workspace}}\oneapi-tbb-2021.12.0-win.zip -DestinationPath ${{github.workspace}}
+  
+    - name: Configure build
+      run: >
+        cmake
+        -B ${{env.BUILD_DIR}}
+        -DCMAKE_INSTALL_PREFIX="${{env.INSTL_DIR}};${{github.workspace}}\hwloc-win64-build-2.10.0;${{github.workspace}}\oneapi-tbb-2021.12.0"
```

Comment:
install prefix should be set to `-DCMAKE_INSTALL_PREFIX="${{env.INSTL_DIR}}"`
what you want to set up here is `-DCMAKE_PREFIX_PATH="tbb_dir;hwloc_dir"

Reply:
done